src/check.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function check(userNum, randomNum) {
```

Comment:
The function should return the number of bulls and cows, not an array with '*' and 'bull' or 'cow'. Consider changing the result format.